src/utils/rtl.js

```diff
@@ -9,6 +9,10 @@ const reSkew = /((^|\s)skew(x|y)?\()\s*(\-?[\d]+)(deg|rad|grad)(,\s*(\-?[\d]+)(d
 export default function rtl(muiTheme) {
   if (muiTheme.isRtl) {
     return (style) => {
```

Comment:
I'm a bit worried about the performance implication of a destructuration for all the style objets.
Alternally, we could be using the second argument of the `prepareStyle` method.
But I don't think that providing an invalid style property has any real downside.